routes/track.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const router = express.Router();
+const Package = require("../db/Models/Package");
+
+router.get("/", async(req, res) =>{
+    res.render("track");
+});
+router.post("/", async(req, res) =>{
+const userEmail = req.body.email;
+const findEmail = await Package.findAll({
+    where: {
```

Comment:
Spacing should be 2

Fix with lint

Reply:
fixed in 1ce2e7aca0bbd21a4f8091041c32c6a311e98dda